SECURITY.md

```diff
@@ -0,0 +1,10 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+All IPython and Jupyter security are handled via security@ipython.org.
+You can find more information on the Jupyter website. https://jupyter.org/security
+
+## Tidelift
```

Comment:
This seems to contradict the previous paragraph that security is handled through security@ipython.org?